client/components/tinymce/plugins/simple-payments/dialog/index.jsx

```diff
@@ -181,11 +181,42 @@ class SimplePaymentsDialog extends Component {
 		return isNumber( this.props.editPaymentId );
 	}
 
+	checkUnsavedForm() {
+		const formIsUnsaved = this.state.activeTab === 'form' && this.props.formIsDirty;
```

Comment:
I'd use `get` if there was a realistic chance that `this.state` is not an object, like if it was coming from somewhere outside the component. But here it's pretty much guaranteed that it's a valid object.